symphony-bdk-core/src/main/java/com/symphony/bdk/core/config/model/BdkRetryConfig.java

```diff
@@ -0,0 +1,24 @@
+package com.symphony.bdk.core.config.model;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class BdkRetryConfig {
+
+    public static final int DEFAULT_MAX_ATTEMPTS = 10;
+    public static final long DEFAULT_INITIAL_INTERVAL_MILLIS = 500L;
+    public static final double DEFAULT_MULTIPLIER = 2;
+    public static final long DEFAULT_MAX_INTERVAL_MILLIS = 5*60*1000;
```

Comment:
with those defaults it looks like (if I understand resilience4j correctly) that the retries are going to be 
500, 1000, 2000, 4000, 8000, 16000, 32000, 64000, 128000, 360000 and that's the 10th attempts so we stop

so do we need a 360000 max interval limit in the end? (see also the comment below on the Exponential interval function)